src/main/java/at/hannibal2/skyhanni/features/bingo/MinionCraftHelper.kt

```diff
@@ -33,10 +34,19 @@ class MinionCraftHelper {
 
     private val config get() = SkyHanniMod.feature.event.bingo
 
-    private val minionNamePattern by RepoPattern.pattern(
-        "bingo.minion.name",
+    private val patternGroup = RepoPattern.group("bingo.minion")
+    private val minionNamePattern by patternGroup.pattern(
+        "name",
         "(?<name>.*) Minion (?<number>.*)"
     )
+    private val isMinionNamePattern by RepoPattern.pattern(
+        "isminionname",
+        ".* Minion (?!Skin).*"
+    )
```

Comment:
this can probably be changed to just be one pattern if we specify what number can be in minionNamePattern then use that pattern for checking isMinionName